requirements.txt

```diff
@@ -1,6 +1,6 @@
 -e git+https://github.com/datastax/python-driver.git@cassandra-test#egg=cassandra-driver
 # Used ccm version is tracked by cassandra-test branch in ccm repo. Please create a PR there for fixes or upgrades to new releases.
--e git+https://github.com/riptano/ccm.git@cassandra-test#egg=ccm
+-e git+https://github.com/aweisberg/ccm.git@cassandra-test#egg=ccm
```

Comment:
Could you ensure that you change this when outstanding changes in upstream `ccm` are merged?

Reply:
fixed